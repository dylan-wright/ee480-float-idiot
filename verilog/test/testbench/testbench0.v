x = 3;
op = `OPi2f;
#10;
$display("%h", z);
x = 16'h4040;
op = `OPf2i;
#10;
$display("%h", z);
x = -10;  //check negative i2f
op = `OPi2f;
#10;
$display("%h", z);
x = 16'hc120; //check negative f2i
op = `OPf2i;
#10;
$display("%h", z);
x = -65535;  //check large negative i2f
op = `OPi2f;
#10;
$display("%h", z);
x = 16'hc77f; //check large negative f2i
op = `OPf2i;
#10;
$display("%h", z);
