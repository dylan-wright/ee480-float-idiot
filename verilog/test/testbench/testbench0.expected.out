4040
0003
c120
000a
c77f
ffff
